shoov/modules/custom/shoov_repository/handlers/shoov_repository_subscribe_user.inc

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains shoov_repository_subscribe_user.
+ */
+
+class shoov_repository_subscribe_user extends views_handler_field_entity {
+
+  /**
+   * {@inheritdoc}
+   */
+  function render($values) {
+    // Get group is being viewed ID.
+    $argument = $this->view->argument;
+    if (empty($argument['og_user_node_target_id'])) {
```

Comment:
I wouldn't change it now, but fyi - this is wrong ;) You are checking an hardcoded field name, but it could have been other names as-well.

Anyway, since this is very specific and not a generic module, we can leave as-is.